e2e-tests/trade-finance-manager/cypress/integration/journeys/case/amendments/amendment-tasks-manual.spec.js

```diff
@@ -0,0 +1,445 @@
+
+    pages.tasksPage.tasks.row(3, 2).status().first().contains('Done');
+
+    pages.tasksPage.tasks.row(3, 1).status().contains('To do');
+    pages.tasksPage.tasks.row(4, 1).status().contains('Cannot start yet');
+    pages.tasksPage.tasks.row(4, 2).status().contains('Cannot start yet');
+    pages.tasksPage.tasks.row(4, 3).status().contains('Cannot start yet');
+    pages.tasksPage.tasks.row(4, 4).status().contains('Cannot start yet');
+    pages.tasksPage.tasks.row(4, 5).status().contains('Cannot start yet');
+
+    pages.tasksPage.tasks.row(3, 1).link().first().click();
+
+    const completeTaskParams1 = {
+      userId,
+      groupId: 3,
+      taskId: 1,
+    };
+    completeTask(completeTaskParams1);
+    cy.url().should('eq', relative(`/case/${dealId}/tasks`));
+    pages.tasksPage.filterRadioAllTasks().click();
+
+    pages.tasksPage.tasks.row(3, 1).status().first().contains('Done');
+
+    pages.tasksPage.tasks.row(4, 1).status().contains('To do');
+    pages.tasksPage.tasks.row(4, 2).status().contains('Cannot start yet');
+    pages.tasksPage.tasks.row(4, 3).status().contains('Cannot start yet');
+    pages.tasksPage.tasks.row(4, 4).status().contains('Cannot start yet');
+    pages.tasksPage.tasks.row(4, 5).status().contains('Cannot start yet');
+  });
+
+  it('should allow you to add UW decision after completing `approve or decline the amendment` task in order', () => {
+    cy.login(PIM_USER_1);
+    cy.visit(relative(`/case/${dealId}/deal`));
+    caseSubNavigation.tasksLink().click();
+    cy.url().should('eq', relative(`/case/${dealId}/tasks`));
+    pages.tasksPage.filterRadioAllTasks().click();
+
+    pages.tasksPage.tasks.row(4, 1).link().first().click();
+
+    const completeTaskParams = {
+      userId,
+      groupId: 4,
+      taskId: 1,
+    };
+    completeTask(completeTaskParams);
+    cy.url().should('eq', relative(`/case/${dealId}/tasks`));
+    pages.tasksPage.filterRadioAllTasks().click();
+
+    pages.tasksPage.tasks.row(4, 1).status().first().contains('Done');
+    pages.tasksPage.tasks.row(4, 2).status().contains('To do');
+    pages.tasksPage.tasks.row(4, 3).status().contains('Cannot start yet');
+    pages.tasksPage.tasks.row(4, 4).status().contains('Cannot start yet');
+    pages.tasksPage.tasks.row(4, 5).status().contains('Cannot start yet');
+
+    pages.tasksPage.tasks.row(4, 2).link().first().click();
+
+    const completeTaskParams1 = {
+      userId,
+      groupId: 4,
+      taskId: 2,
+    };
+    completeTask(completeTaskParams1);
+    cy.url().should('eq', relative(`/case/${dealId}/tasks`));
+    pages.tasksPage.filterRadioAllTasks().click();
+
+    pages.tasksPage.tasks.row(4, 2).status().contains('Done');
+    pages.tasksPage.tasks.row(4, 3).status().contains('To do');
+    pages.tasksPage.tasks.row(4, 4).status().contains('Cannot start yet');
+    pages.tasksPage.tasks.row(4, 5).status().contains('Cannot start yet');
+
+    // cy.login(UNDERWRITER_MANAGER_1);
```

Comment:
do we need this commented code?

Reply:
yes, have left todo comment.  Needed for when UW managers decision is dependent on tasks